controllers/cart-items-controllers.js

```diff
@@ -0,0 +1,39 @@
+const {
+    getAllCartItems,
+    selectCartItemsByCartId
+} = require('../services/cart-items-service');
+
+const getCartItemsRoute = (server) => {
+    server.route({
+        path: '/cart-items',
+        method: 'GET',
+        handler: (request, h) => {
+            return getAllCartItems();
+        }
+    });
+};
+
+const getCartItemsByItemIdRoute = (server) => {
+    server.route({
+        path: '/cart-items/{cartItemId}',
+        method: 'GET',
+        handler: (request, h) => {
+            const cartItems = getCartItemsByItemId(request.params.cartItemId);
```

Comment:
const cartItem = getCartItemByItemId(request.params.cartItemId);

we are requesting a single resource by single resource id